docs/development-environment.md

```diff
@@ -124,14 +124,14 @@ You'll need all the tools below to work in the Jetpack monorepo.
 
 		Composer can be installed using [Homebrew](https://brew.sh/). If you don't have Homebrew, install it with
 
-		```sh
-	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+		```
+		sh /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
I think we can leave `sh` with the fenced code block indicators, it's meant to be used to add syntax highlighting. It doesn't work properly in this case because of the indentation issue. This should work:

```suggestion
		```sh
		/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```